src/Driver/Driver.jl

```diff
@@ -145,6 +149,9 @@ function parse_commandline(custom_settings)
         metavar = "<number>"
         arg_type = Int
         default = -1
+        "--fix-rng-seed"
+        help = "set RNG seed to a fixed value for reproducibility"
+        action = :store_true
```

Comment:
```suggestion
        default = true
```
?

Reply:
It's a no-argument... argument. That is, you just add `--fix-rng-seed`, not `--fix-rng-seed=10` or anything. So when the argument is found, this action stores `true` in the relevant `Dict` entry. See [here](https://carlobaldassi.github.io/ArgParse.jl/stable/arg_table/#Available-actions-and-nargs-values-1) if you want to know the details!